filebeat/module/tidb/_meta/config.reference.yml

```diff
@@ -0,0 +1,113 @@
+#- module: tidb
+#
+#  slowlog:
+#    enabled: true
+#    # Set custom paths for the log files. If left empty,
+#    # Filebeat will choose the default TiUP cluster path.
+#    #var.paths:
```

Comment:
As a reference, should we supply as much information as possible? The current reference comments are just not well-informed enough. User must find another document to know how to write it.

Reply:
Fixed.